src/Component/App.js

```diff
@@ -1,14 +1,249 @@
+        This project is based on facebook create react app and try to follow the community guides.
+      </Item>
+
+      <Item>
+        <Label
+          target="_blank"
+          href="https://www.styled-components.com/"
+        >
+          Styled Components
+        </Label>
+        Making design is often very painfull: How to organise classes ? How to make reusable components ?
+        Should i use id ? Does my classe works with this component ? Where is my fu***** stylesheet ? ...
+        <br />
+        <br />
+        Styled components is a very popular and handy tools allowing you make
+        designs with (wait for it...) Component.
+        <br/>
+        <br />
+        You will love it!
+      </Item>
+
+      <Item>
+        <Label
+          target="_blank"
+          href="https://reacttraining.com/react-router/web/guides/quick-start"
+        >
+          React Router
+        </Label>
+        Any web application need some routing, allowing people to navigate into your app.
+        React Router is the most popular routing tools for react.
+      </Item>
+
+      <Item>
+        <Label
+          target="_blank"
+          href="https://redux.js.org/"
+        >
+          Redux
+        </Label>
+        It uses Redux as a default data layer allowing purity and global state management.
+      </Item>
+
+      <Item>
+        <Label
+          target="_blank"
+          href="https://github.com/Djeg/ramduck-redux#readme"
+        >
+          Ramduck Redux
+        </Label>
+        This is one of our handy tool making reducers and actions very powerfull. It simplyfies
+        and currify your reducers and actions, making your code very consice and easilty testable.
+      </Item>
+
+      <Item>
+        <Label
+          target="_blank"
+          href="https://redux-observable.js.org/"
+        >
+          Redux Observable
+        </Label>
+        Every frontend application has impure functions making effects.
+        This project comes with the most powerfull reactive tooling for javascript powered
+        by <Link href="https://rxjs.dev/" target="_blank">RxJS</Link> allowing you to
+        manage those effects.
+      </Item>
+
+      <Item>
+        <Label
+          target="_blank"
+          href="https://ramdajs.com"
+        >
+          Ramda
+        </Label>
+        Ramda is a very powerfull set of functions helping you building "functional" application.
+      </Item>
+
+      <Item>
+        <Label
+          target="_blank"
+          href="https://evilsoft.github.io/crocks/"
+        >
+          Crocks
+        </Label>
+        You may want to make impure functions without all the complexity of an observable. Crocks
+        is a complete set of well known and tested Algebraic Data Types. Use it to wrap easily
+        any impure operations and use the power of Categories to map, chain, ap ... any effects
+      </Item>
+    </Menu>
+  </Fragment>
+
+
+export const BlackStrip = styled.div`
```

Comment:
These styles should belong in ad edicated CSS file IMHO, to avoid very long files suche as this one.

Reply:
Ive play a lot with `styled-components`. I was first not confortable at all with the concept but gosh, it's sooooo more maintainable and brings css into your cadebase directly!

I've say in the homepage:

```
        Making design is often very painfull: How to organise classes ? How to make reusable components ?
        Should i use id ? Does my classe works with this component ? Where is my fu***** stylesheet ? ...
        <br />
        <br />
        Styled components is a very popular and handy tools allowing you make
        designs with (wait for it...) Component.
        <br/>
        <br />
        You will love it!

```

Reply:
I agree too, if you still want to use `styled-components`, consider to put these styled components in a dedicated file ;)

Reply:
I don't agree. In many cases you may want to have a UI sort of file wich will contains some styled components for a given widget or anything. I agree. But here every component i have is directly attach to my **domain**. Why putting what is your domain outside of your domain ? you can still import those components `import { BlackStrip } from './App'` and we stay **closest as possible** to the domain :). I don't think it's needed here to separate. The application is a single page so ne no need plural files

Reply:
Besides, this repository is a bootstrapper, you will have to delete all this bootstrapping code. Here you just have to clean the `App` and start over. If i separate it will 2 files to delete. And i really don't see the point to separate here ?

Reply:
I'm not sure about all of this, didn't we said we shouldn't bloat this app? Maybe just a part in the main readme telling people styled components are great and show an example of how you can make it work within this very project could be way better?